SPECS/libyang/libyang.spec

```diff
@@ -0,0 +1,103 @@
+# valgrind finds invalid writes in libcmocka on arm and power
+# see bug #1699304 for more information
+%ifarch %{arm} ppc64le
+%global run_valgrind_tests OFF
+%else
+%global run_valgrind_tests ON
+%endif
+Summary:        YANG data modeling language library
+Name:           libyang
+Version:        2.0.231
+Release:        1%{?dist}
+License:        BSD-3-Clause
```

Comment:
```suggestion
License:        BSD
```

Reply:
Sure, but OOI why? I see a bunch of specs in mariner with BSD-3-Clause as the license, and assumed it was better to have a more specific declared license.

Reply:
Fair question. We try to stick to [the "Short names" Fedora is using for its specs](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses). There are some leftovers from before we were more strick here and sometimes things slip through the cracks, but in general we try to follow that convention.